tensorflow/core/kernels/mkl/mkl_conv_ops.cc

```diff
@@ -1278,6 +1507,20 @@ class MklFusedConvOp
     }
   }
 
+  virtual void ComputeBNScale(OpKernelContext* context, float epsilon,
```

Comment:
Why is this a virtual function?

Reply:
Good question. The ComputBNScale is only applicable to the MklFusedConvOp, but with the current design of mkl_conv_ops, the calling of ComputeBNScale is common to all the subclasses. I need to make this function virtual, so the computation is only invoked for MklFusedConvOp, and for the rest, no computation is invoked.

Reply:
Thank you for the clarification! Then I think the signature of this function should be
```c++
void ComputeBNScale(...) override {...}
```
instead, since it's in a derived class. I'll fix this internally.

Reply:
thanks!